creator-node/scripts/updateContentBlacklist.js

```diff
@@ -0,0 +1,157 @@
+const axios = require('axios')
```

Comment:
let's double check w/ dheeraj. maybe we should put this in service commands.

Reply:
@dmanjunath